jit/lib/setupContext.js

```diff
@@ -229,7 +229,7 @@ function trackModified(files) {
   for (let file of files) {
     if (!file) continue
 
-    let pathname = url.parse(file).pathname
+    let pathname = url.parse(file).href
```

Comment:
I will try that change and make the appropriate updates to the PR
meanwhile I would like to ask if this is intended to run outside of a node environment. under what circumstances would there ever be a hash/query params. it would really help to understand how I could have caught this use case earlier

Reply:
ok, now the failed tests make more sense
I'll get right on those changes

Reply:
Yeah the only time it happens is in some build tools like Vite that proxy URLs to the file system and add query string information. For example in Vite it might make a request for a Vue file just to get the CSS portion using a path that's sort of like this if I remember correctly:

`http://localhost:3000/@/src/App.vue?lang=css`

That gets converted to a file name when passed in to PostCSS and the query string is preserved, so we need to chop it off.